templates/Segment_Image_Data.ipynb

```diff
@@ -288,19 +285,6 @@
     "                                             fovs=fovs,\n",
     "                                             batch_size=5)"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# save extracted data as csv for downstream analysis\n",
-    "cell_table_size_normalized.to_csv(os.path.join(single_cell_dir, 'cell_table_size_normalized.csv'),\n",
-    "                                 index=False)\n",
-    "cell_table_arcsinh_transformed.to_csv(os.path.join(single_cell_dir, 'cell_table_arcsinh_transformed.csv'),\n",
-    "                                     index=False)"
```

Comment:
Hey guys, was reviewing your branch to see what possible things needed to be added for the notebook testing suite and couldn't help but notice this. This last cell was probably removed by accident; you'll need to put it back in. You can refer to the code in the master branch to see what it looks like.